sirepo/package_data/static/en/privacy.html

```diff
@@ -188,7 +189,7 @@ <h4>Contact Information</h4>
               <li><a href="mailto:support@sirepo.com?subject=Web+Contact">Contact Us</a></li>
               <li><a href="http://radiasoft.net">RadiaSoft Home</a></li>
```

Comment:
old issue: https://radiasoft.net. And, no need for trailing "/", i.e. just href="https://radiasoft.net"